Asciidoc fails to start with `com/intellij/ui/jcef/JBCefApp` not found on 2026.2 Beta

## Observed vs. expected behavior

<!--
    Describe the issue you are experiencing:
    What behaviour do you see in the plugin or the IDE?
    What is the expected behaviour?
-->
Should properly declare jcef, I think that's the issue. But unsure.


## Steps to reproduce

<!--
    Describe the steps to reproduce the issue.
    Please provide a minimal Asciidoctor example.
    Consider adding screenshots or video.
-->

Run IntelliJ 2026.2 Beta, install Asciidoc, restart. An error shows about calls not found.

```
com.intellij.diagnostic.PluginException: com/intellij/ui/jcef/JBCefApp [Plugin: org.asciidoctor.intellij.asciidoc] [Plugin: org.asciidoctor.intellij.asciidoc]
	at com.intellij.ide.startup.impl.StartupManagerImplKt$launchActivity$1.invokeSuspend(StartupManagerImpl.kt:542)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:610)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runDefaultDispatcherTask(CoroutineScheduler.kt:1194)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:906)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:775)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:762)
Caused by: com.intellij.diagnostic.PluginException: com/intellij/ui/jcef/JBCefApp [Plugin: org.asciidoctor.intellij.asciidoc]
	at com.intellij.serviceContainer.ServiceInstanceInitializer.createInstance$suspendImpl(ServiceInstanceInitializer.kt:58)
	at com.intellij.serviceContainer.ServiceInstanceInitializer.createInstance(ServiceInstanceInitializer.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invokeSuspend(LazyInstanceHolder.kt:181)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invoke(LazyInstanceHolder.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invoke(LazyInstanceHolder.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndspatched(Undispatched.kt:67)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:43)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:165)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invokeSuspend(LazyInstanceHolder.kt:179)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invoke(LazyInstanceHolder.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invoke(LazyInstanceHolder.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startCoroutineUndispatched(Undispatched.kt:20)
	at kotlinx.coroutines.CoroutineStart.invoke(CoroutineStart.kt:360)
	at kotlinx.coroutines.AbstractCoroutine.start(AbstractCoroutine.kt:134)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.launch(Builders.common.kt:53)
	at kotlinx.coroutines.BuildersKt.launch(Unknown Source)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.initialize(LazyInstanceHolder.kt:164)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.access$initialize(LazyInstanceHolder.kt:29)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.tryInitialize(LazyInstanceHolder.kt:154)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstance(LazyInstanceHolder.kt:112)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstanceInCallerContext$suspendImpl(LazyInstanceHolder.kt:104)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstanceInCallerContext(LazyInstanceHolder.kt)
	at com.intellij.serviceContainer.ComponentManagerImplKt$doGetOrCreateInstanceBlocking$1.invokeSuspend(ComponentManagerImpl.kt:1698)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:263)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:112)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$BuildersKt__BuildersKt(Builders.kt:87)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlockingWithParallelismCompensation(Builders.kt:63)
	at kotlinx.coroutines.BuildersKt.runBlockingWithParallelismCompensation(Unknown Source)
	at kotlinx.coroutines.internal.intellij.IntellijCoroutines.runBlockingWithParallelismCompensation(intellij.kt:48)
	at com.intellij.util.IntelliJCoroutinesFacade.runBlockingWithParallelismCompensation(IntelliJCoroutinesFacade.kt:35)
	at com.intellij.serviceContainer.ComponentManagerImplKt.runBlockingInitialization$lambda$0(ComponentManagerImpl.kt:1829)
	at com.intellij.openapi.progress.ContextKt.prepareThreadContext$lambda$0(context.kt:96)
	at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:295)
	at com.intellij.openapi.progress.ContextKt.prepareThreadContext(context.kt:95)
	at com.intellij.serviceContainer.ComponentManagerImplKt.runBlockingInitialization(ComponentManagerImpl.kt:1814)
	at com.intellij.serviceContainer.ComponentManagerImplKt.doGetOrCreateInstanceBlocking(ComponentManagerImpl.kt:1697)
	at com.intellij.serviceContainer.ComponentManagerImplKt.getOrCreateInstanceBlocking(ComponentManagerImpl.kt:1686)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:798)
	at com.intellij.serviceContainer.ComponentManagerImpl.getService(ComponentManagerImpl.kt:736)
	at org.asciidoc.intellij.settings.AsciiDocApplicationSettings.getInstance(AsciiDocApplicationSettings.java:39)
	at org.asciidoc.intellij.activities.AsciiDocPluginUpdateActivity.lambda$execute$2(AsciiDocPluginUpdateActivity.java:37)
	at com.intellij.util.JavaCoroutines.suspendJava(JavaCoroutines.kt:36)
	at org.asciidoc.intellij.activities.AsciiDocPluginUpdateActivity.execute(AsciiDocPluginUpdateActivity.java:36)
	at com.intellij.ide.startup.impl.StartupManagerImplKt$launchActivity$1.invokeSuspend(StartupManagerImpl.kt:536)
	... 7 more
Caused by: java.lang.NoClassDefFoundError: com/intellij/ui/jcef/JBCefApp
	at org.asciidoc.intellij.editor.jcef.AsciiDocJCEFHtmlPanelProvider.isAvailable(AsciiDocJCEFHtmlPanelProvider.java:61)
	at org.asciidoc.intellij.settings.AsciiDocPreviewSettings.<init>(AsciiDocPreviewSettings.java:40)
	at org.asciidoc.intellij.settings.AsciiDocPreviewSettings.<clinit>(AsciiDocPreviewSettings.java:28)
	at org.asciidoc.intellij.settings.AsciiDocApplicationSettings$State.<init>(AsciiDocApplicationSettings.java:97)
	at org.asciidoc.intellij.settings.AsciiDocApplicationSettings.<init>(AsciiDocApplicationSettings.java:29)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt.instantiate$lambda$0(instantiate.kt:48)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt.instantiate$lambda$3$0(instantiate.kt:294)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt.withStoredTemporaryContext(instantiate.kt:310)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt.instantiate(instantiate.kt:293)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt.instantiate(instantiate.kt:45)
	at com.intellij.serviceContainer.ServiceInstanceInitializer.createInstance$suspendImpl(ServiceInstanceInitializer.kt:33)
	... 53 more
Caused by: java.lang.ClassNotFoundException: com.intellij.ui.jcef.JBCefApp PluginClassLoader(plugin=PluginMainDescriptor(name=AsciiDoc, id=org.asciidoctor.intellij.asciidoc, version=0.45.5, isBundled=false, path=~/Library/Application Support/JetBrains/IntelliJIdea2026.2/plugins/asciidoctor-intellij-plugin), packagePrefix=null, state=active, parents=ContentModuleDescriptor(id=intellij.java.bookmarks, package=com.intellij.java.bookmarks) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.grazie.yaml, package=com.intellij.grazie.ide.language.yaml) <- PluginMainDescriptor(name=Natural Languages, id=tanvd.grazi, version=262.8665.81, package=com.intellij.grazie, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/grazie), ContentModuleDescriptor(id=intellij.grazie.xml, package=com.intellij.grazie.ide.language.xml) <- PluginMainDescriptor(name=Natural Languages, id=tanvd.grazi, version=262.8665.81, package=com.intellij.grazie, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/grazie), ContentModuleDescriptor(id=intellij.grazie.markdown, package=com.intellij.grazie.ide.language.markdown) <- PluginMainDescriptor(name=Natural Languages, id=tanvd.grazi, version=262.8665.81, package=com.intellij.grazie, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/grazie), ContentModuleDescriptor(id=intellij.grazie.json, package=com.intellij.grazie.ide.language.json) <- PluginMainDescriptor(name=Natural Languages, id=tanvd.grazi, version=262.8665.81, package=com.intellij.grazie, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/grazie), ContentModuleDescriptor(id=intellij.libraries.groovy, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Groovy Scripting, id=com.intellij.groovy.scripting, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/groovy-scripting), ContentModuleDescriptor(id=intellij.platform.vcs.log.graph) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.xml.emmet) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.ssh.core, loadingRule=REQUIRED) <- PluginMainDescriptor(name=SSH, id=intellij.ssh.plugin, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/platform-ssh-plugin), ContentModuleDescriptor(id=intellij.platform.ssh, loadingRule=REQUIRED) <- PluginMainDescriptor(name=SSH, id=intellij.ssh.plugin, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/platform-ssh-plugin), ContentModuleDescriptor(id=intellij.platform.ssh.attach) <- PluginMainDescriptor(name=SSH, id=intellij.ssh.plugin, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/platform-ssh-plugin), ContentModuleDescriptor(id=intellij.platform.vcs.log) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.smRunner, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Test Runner, id=intellij.testRunner.plugin, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/platform-testRunner-plugin), ContentModuleDescriptor(id=intellij.platform.structureView) <- PluginMainDescriptor(name=Structure View, id=intellij.structureView.plugin, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/platform-structureView-plugin), ContentModuleDescriptor(id=intellij.platform.todo) <- PluginMainDescriptor(name=TODO Comments, id=intellij.todo.plugin, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/platform-todo-plugin), ContentModuleDescriptor(id=intellij.platform.bookmarks, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Bookmarks Manager, id=intellij.bookmarks.plugin, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/platform-bookmarks-plugin), ContentModuleDescriptor(id=intellij.platform.vcs.dvcs) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.structuralSearch) <- PluginMainDescriptor(name=Structural Search, id=intellij.structuralSearch.plugin, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/platform-structuralSearch-plugin), ContentModuleDescriptor(id=intellij.relaxng) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.vcs.impl) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.vcs.log.impl) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.vcs.dvcs.impl) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.spellchecker) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.json.backend, loadingRule=REQUIRED) <- PluginMainDescriptor(name=JSON, id=com.intellij.modules.json, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/json), ContentModuleDescriptor(id=intellij.java.backend, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.spellchecker.xml) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.scriptDebugger.protocolReaderRuntime) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.scriptDebugger.backend) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.scriptDebugger.ui) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.tasks) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.tasks.impl) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.collaborationTools.auth.base) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.collaborationTools.auth) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.collaborationTools) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.langInjection) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/lib), ContentModuleDescriptor(id=intellij.java.syntax, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.frontback.psi, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.frontback.psi.impl, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.psi, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.psi.impl, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.frontback.impl, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.indexing, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.platform.uast, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.platform.uast.ide, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.uast, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.compiler, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.analysis, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.execution, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.remoteServers) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.jvm.analysis, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.jsp, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.debugger, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.codeserver.core, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.codeserver.highlighting, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.jsp.spi, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.indexing.impl, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.analysis.impl, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.structureView, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.impl, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.xml.dom.impl) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.impl.refactorings, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.compiler.impl, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.typeMigration, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.impl.inspections, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.jvm.analysis.refactoring, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.jvm.analysis.quickFix, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.uiDesigner, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.manifest, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.ui, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.execution.impl) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.debugger.impl) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.debugger.memory.agent, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.remoteServers.impl) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.uast.ide, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.platform.jps.build, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.platform.jps.build.dependencyGraph, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.platform.jps.build.javac.rt, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.guiForms.compiler, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.compiler.antTasks, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.compiler.instrumentationUtil, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.compiler.instrumentationUtil.java8, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.backend, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.frontend, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.backend.split) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.debugger.impl.shared, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.execution.impl.shared, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.structuralSearch) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.featuresTrainer) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.performancePlugin) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.unscramble) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.vcs) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.terminal.shared) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.terminal.backend) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.terminal.frontend) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.jvm.analysis.impl) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.debugger.impl.frontend) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.debugger.impl.backend) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.execution.impl.frontend) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.execution.impl.backend) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.ultimate.icons) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.langInjection) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.langInjection.jps) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.duplicates) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.duplicates.analysis) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.duplicatesDetection) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.jam.impl) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.jam) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.profiler.ultimate) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.debugger.jvm.advanced.java) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.debugger.logpoints.shared) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.debugger.logpoints.backend) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.debugger.logpoints.frontend) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.debugger.jvm.advanced.java.logpoints) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.debugger.logpoints.java) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.copyright) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.libraries.test.discovery, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.json, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=JSON, id=com.intellij.modules.json, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/json), ContentModuleDescriptor(id=intellij.json.backend, loadingRule=REQUIRED) <- PluginMainDescriptor(name=JSON, id=com.intellij.modules.json, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/json), ContentModuleDescriptor(id=intellij.json.networknt.wrapper, loadingRule=REQUIRED) <- PluginMainDescriptor(name=JSON, id=com.intellij.modules.json, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/json), ContentModuleDescriptor(id=intellij.json.backend.regexp) <- PluginMainDescriptor(name=JSON, id=com.intellij.modules.json, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/json), ContentModuleDescriptor(id=intellij.json.structureView) <- PluginMainDescriptor(name=JSON, id=com.intellij.modules.json, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/json), ContentModuleDescriptor(id=intellij.json.syntax, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=JSON, id=com.intellij.modules.json, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/json), ContentModuleDescriptor(id=intellij.yaml.backend) <- PluginMainDescriptor(name=YAML, id=org.jetbrains.plugins.yaml, version=262.8665.81, package=org.jetbrains.yaml, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/yaml), ContentModuleDescriptor(id=intellij.yaml, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=YAML, id=org.jetbrains.plugins.yaml, version=262.8665.81, package=org.jetbrains.yaml, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/yaml), ContentModuleDescriptor(id=intellij.grazie.core, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Natural Languages, id=tanvd.grazi, version=262.8665.81, package=com.intellij.grazie, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/grazie), ContentModuleDescriptor(id=intellij.grazie.java) <- PluginMainDescriptor(name=Natural Languages, id=tanvd.grazi, version=262.8665.81, package=com.intellij.grazie, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/grazie), ContentModuleDescriptor(id=intellij.grazie.properties) <- PluginMainDescriptor(name=Natural Languages, id=tanvd.grazi, version=262.8665.81, package=com.intellij.grazie, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/grazie), ContentModuleDescriptor(id=intellij.libraries.ai.grazie.spell.gec.engine.local, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Natural Languages, id=tanvd.grazi, version=262.8665.81, package=com.intellij.grazie, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/grazie), ContentModuleDescriptor(id=intellij.sh.core, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Shell Script, id=com.jetbrains.sh, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/sh-plugin), ContentModuleDescriptor(id=intellij.sh.backend) <- PluginMainDescriptor(name=Shell Script, id=com.jetbrains.sh, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/sh-plugin), ContentModuleDescriptor(id=intellij.sh.terminal) <- PluginMainDescriptor(name=Shell Script, id=com.jetbrains.sh, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/sh-plugin), ContentModuleDescriptor(id=intellij.sh.copyright) <- PluginMainDescriptor(name=Shell Script, id=com.jetbrains.sh, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/sh-plugin), ContentModuleDescriptor(id=intellij.sh.markdown) <- PluginMainDescriptor(name=Shell Script, id=com.jetbrains.sh, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/sh-plugin), PluginMainDescriptor(name=YAML, id=org.jetbrains.plugins.yaml, version=262.8665.81, package=org.jetbrains.yaml, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/yaml), PluginMainDescriptor(name=Natural Languages, id=tanvd.grazi, version=262.8665.81, package=com.intellij.grazie, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/grazie), PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/java), PluginMainDescriptor(name=JSON, id=com.intellij.modules.json, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/json), PluginMainDescriptor(name=Shell Script, id=com.jetbrains.sh, version=262.8665.81, isBundled=true, path=~/Applications/IntelliJ IDEA 2026.2 Beta.app/Contents/plugins/sh-plugin), )
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.kt:154)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:490)
	... 64 more
```



## Environment

<!--
    This information helps the maintainers to reproduce the issue in their environment.
    Go to: File -> Settings -> Plugins -> AsciiDoc
    Type in the version number you see
-->

This fails too with

```
java.lang.NoClassDefFoundError: com/intellij/ui/jcef/JBCefApp
	...
```


JCEF plugin is installed
<img width="520" height="83" alt="Image" src="https://github.com/user-attachments/assets/7064b0b8-70eb-4479-a62a-0fbfc37b9835" />

Plugin Version: 0.45.5

<!--
    Go to: Help -> About
    Use the "Copy and Close" button and paste the information below.
-->
IntelliJ Details: 2025.2 Beta


----


Somewhat relate to https://github.com/asciidoctor/asciidoctor-intellij-plugin/issues/2026

--- a/CHANGELOG.adoc
+++ b/CHANGELOG.adoc
@@ -9,7 +9,7 @@ This document provides a high-level view of the changes introduced by release.
 
 === 0.45.6
 
-...
+- Declare dependency on the JCEF module so the plugin loads on 2026.2 where JCEF became a separate plugin (#2073)
 
 === 0.45.5
 

--- a/src/main/resources/META-INF/plugin.xml
+++ b/src/main/resources/META-INF/plugin.xml
@@ -15,6 +15,8 @@
   <idea-version since-build="253.25908"/> <!-- 2025.1 EAP, also change build.gradle.kts -->
   <!-- please see https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html -->
   <depends>com.intellij.modules.lang</depends>
+  <!-- JCEF is a module alias of the core IDE up to 2026.1, and a separate plugin starting with 2026.2 (#2073) -->
+  <depends>com.intellij.modules.jcef</depends>
   <depends optional="true" config-file="org.asciidoctor.intellij.asciidoc-intellilang.xml">org.intellij.intelliLang</depends>
   <depends optional="true" config-file="org.asciidoctor.intellij.asciidoc-java.xml">com.intellij.modules.java</depends>
   <depends optional="true" config-file="org.asciidoctor.intellij.asciidoc-json.xml">com.intellij.modules.json</depends>

